src/app/core/services/issue-comment.service.ts

```diff
@@ -24,6 +25,14 @@ export class IssueCommentService {
     return of(this.comments.get(issueId));
   }
 
+  getGithubComments(issueId: number, isIssueReloaded: boolean): Observable<GithubComment[]> {
+    return this.githubService.fetchIssueComments(issueId).pipe(
+      map(rawJsonDataArray => rawJsonDataArray.map(rawJsonData => <GithubComment> {
+        ...rawJsonData
+      }))
```

Comment:
Can pipe this in github.service? So `githubService.fetchIssueComments` should return `Observable<GithubComment[]>`.

Reply:
Sure 👍  But I will create it as a separate function first to prevent anything else from breaking?